apps/ssr-test-app/README.md

```diff
@@ -0,0 +1,29 @@
+# SSR Test App
+
+The SSR Test App is a React and NextJS app that serves to ensure any future changes to teams-js do not break server-side rendering capabilities. As it is included in the apps workspace, it will also be built when building teams-js.
+If there are any changes made to teams-js that should break server-side rendering capabilities, the build should fail when it attempts to build the SSR Test App.
+
+# Running the Test App on its own
+
+In order to run the SSR Test App on its own, please follow the following steps
+
+```
+cd {monorepo root}
+
+// Ensuring you have installed and built the Teams JavaScript client SDK
+yarn install
+yarn build
+
+yarn start-ssr-app
+```
+
+or if you have already built the Teams JavaScript client SDK and would like to build and run directly from the project directory ssr-test-app, simply `yarn build` and `yarn start` there.
+
+### Note
+
+Running the SSR Test App locally utilizes an http connection. In order to run the app with local host SDK(Orange), an https connection is required. ngrok can be used to generate an https connection.
```

Comment:
hmm, could you clarify what is local host sdk(orange), is it the sdk, or orange app?
Also, curious, is the reason of ngrok need is because we want to run ssr app in a different port (since default 4000 might be occupied assuming orange above)